reduxfx-view/src/main/java/com/netopyr/reduxfx/vscenegraph/impl/patcher/NodeUtilities.java

```diff
@@ -191,7 +190,7 @@ public static Object getProperty(Object obj, String name) {
                                 .computeIfAbsent("dialog", key -> new HashMap<>())
                 ),
 
-                Case($(), Collections.emptyMap())
+                Case($(), new HashMap<>())
```

Comment:
I don't think that this is the proper fix, though returning an empty map at this point is also not correct.

We need to store internal data with the SceneGraph, e.g. to unregister listeners. Depending on the type of node we are dealing with, we store it slightly differently. But in the end we always have a HashMap where we can put everything we may need later. For this functionality to work correctly, we need to be able to retrieve the HashMap that we generated previously. If we simply return a new empty HashMap, out data is lost.

I am not really sure yet, what we should do in the case of TreeItem. We could probably misuse the value-property. Instead of the actual value, we could store a tuple with the value and a HashMap. Another option would be a global map that stores the HashMaps for all possible TreeItems. We just have to make sure that we do not create a memory leak. Or maybe we have to rethink the whole mechanism. It worked well as long as the node were just JavaFX Nodes, but it becomes more and more of a problem now.